src/java/org/apache/cassandra/index/sai/disk/v1/MemtableIndexWriter.java

```diff
@@ -184,4 +179,43 @@ private long flush(PrimaryKey minKey,
 
         return numRows;
     }
+
+    private void flushVectorIndex(long startTime, Stopwatch stopwatch) throws IOException
+    {
+        SegmentMetadata.ComponentMetadataMap metadataMap = memtable.writeDirect(indexDescriptor, indexContext, rowMapping::get);
+
+        completeIndexFlush(rowMapping.size(), startTime, stopwatch);
+
+        SegmentMetadata metadata = new SegmentMetadata(0,
+                                                       rowMapping.size(),
+                                                       0,
+                                                       rowMapping.maxSSTableRowId,
+                                                       rowMapping.minKey,
+                                                       rowMapping.maxKey,
+                                                       ByteBufferUtil.bytes(0), // VSTODO by pass min max terms for vectors
```

Comment:
are these TODOs still relevant?

Reply:
It isn't really a TODO any more but we could eventually tidy this so we could just not pass anything.